### DISCUSSION TITLE
Failed to export project meta to zip and download [Apr 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1904] [Discord Threads]
> 🙋‍♂️ **YG**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### lets track here - https://github.com/nocodb/nocodb/issues/1755  
#  
> 👨‍🚀 **wingkwong**  
#### at the same time, you may dump `NC_DB` if you want to download the metadata  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/965768176212279408)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
